src/common/modules/Logger.js

```diff
@@ -13,6 +13,36 @@ const MESSAGE_LEVEL_NAME = Object.freeze({
     [MESSAGE_LEVEL.SUCCESS]: "SUCCESS"
 });
 
+/**
+ * Checks if value is a object.
+ *
+ * @function
+ * @param  {*} value
+ * @returns {boolean}
+ */
+function isObject(value){
+    const type = typeof value;
+    return value != null && (type === "object" || type === "function");
+}
+
+/**
+ * Copy nested objects to ensure a properly output.
+ *
+ * @function
+ * @param  {...*} args
+ * @returns {Object}
+ */
+function verifyArgs(args) {
+    for(const key in args) {
```

Comment:
Remember, `args` is an array. As such you do not need to check `hasOwnProperty` and may better use `for... of`. But I also always confuse it.

Here is the relevant explanation: https://stackoverflow.com/questions/33553944/es6-does-for-of-get-prototype-values-how-to-check-hasownproperty#33554329